src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023visualstimulusinterface.py

```diff
@@ -0,0 +1,121 @@
+from typing import Optional
+import h5py
+from neuroconv import BaseDataInterface
+from neuroconv.utils import FilePathType, FolderPathType
+from pynwb import NWBFile
+from pynwb.epoch import TimeIntervals
+from pathlib import Path
+from natsort import natsorted
+from roiextractors.extractors.tiffimagingextractors.scanimagetiff_utils import extract_timestamps_from_file
+import numpy as np
+
+
+def h5py_to_dict(item):
+    if isinstance(item, h5py.Group):
+        group_dict = {}
+        for key, subitem in item.items():
+            group_dict[key] = h5py_to_dict(subitem)
+        return group_dict
+    elif isinstance(item, h5py.Dataset):
+        return item[()]
+    else:
+        return None
+
+
+def match_field_description(field_name):
+    field_description = {
+        "orientation": "orientation of drifting grating in degrees (0-360)",
+        "size_vdeg": "size of stimulus in visual degrees, 1d of length n trials",
+        "contrast": "contrast of gratings, values between 0-1",
+        "location": "(X,Y) location on screen in visual degrees",
+        "vis_orientation_tuning_example": "Visual Orientation Tuning",
+        "vis_orientation_tuning": "Visual Orientation Tuning",  # TODO add a more descriptive text
+        "vis_retinotopy_example": "Retinotopy",
+        "vis_retinotopy": "Retinotopy",  # TODO add a more descriptive text
+        "vis_simple_example": "Simple Visual Stimuls",
+        "vis_simple": "Simple Visual Stimuls",  # TODO add a more descriptive text
+    }
+    if field_name in field_description.keys():
+        return field_description[field_name]
+    else:
+        return None
```

Comment:
This could be achieved with a simple dict:

```
field_description_mapping = dict(orientation="the desc for orientation", ...)
field_description_mapping.get(field_name)
```
when field_name is not in dict .get returns None

Reply:
Right :sweat_smile:
Fixed this